eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/SubmittedVariantAccessioningService.java

```diff
@@ -152,12 +152,12 @@ public void deprecate(Long accession, String reason)
     }
 
     @Override
-    public void merge(Long accession, Long accession1, String reason)
+    public void merge(Long accessionOrigin, Long mergeInto, String reason)
```

Comment:
Suggest renaming to mergeTarget to go along with accessionOrigin...

Reply:
it's not that `mergeInto` is much better, but it is used consistently across the project and the accession library, and accessionTarget sounds ambiguous to me (is it the target accession that you want to merge into another one? no)